tox.ini

```diff
@@ -4,7 +4,7 @@ envlist=py27,py33,py34,static_check
 
 [testenv]
 deps=
-    pytest==2.6.4
+    pytest==2.7.0
```

Comment:
IMO, == is ok. That the py.test version we test against and our logic should not fail when py.test pplz release a new, non-compatible version.